reposerver/repository/repository_test.go

```diff
@@ -985,23 +962,6 @@ func TestListApps(t *testing.T) {
 	assert.Equal(t, expectedApps, res.Apps)
 }
 
-func TestGetAppDetailsHelm(t *testing.T) {
-	service := newService("../..")
-
-	res, err := service.GetAppDetails(context.Background(), &apiclient.RepoServerAppDetailsQuery{
-		Repo: &argoappv1.Repository{},
-		Source: &argoappv1.ApplicationSource{
-			Path: "./util/helm/testdata/helm2-dependency",
```

Comment:
Instead of deleting this test entirely, can you just point it to `../../util/helm/testdata/dependency`?